.github/workflows/image.yml

```diff
@@ -85,7 +85,37 @@ jobs:
         if: ${{ github.event_name == 'pull_request' }}
         run: |
           # start nydus-snapshotter
-          docker run -d --device /dev/fuse --cap-add SYS_ADMIN --security-opt apparmor:unconfined -e CONTAINERD_ROOT=/var/lib/containerd-test -v /var/lib/containerd-test:/var/lib/containerd-test:shared -v ~/.docker/config.json:/root/.docker/config.json:shared ${{ env.REGISTRY }}/${{ env.ORGANIZATION }}/nydus-snapshotter:local
+          docker run -d --name snapshotter --device /dev/fuse --cap-add SYS_ADMIN --security-opt apparmor:unconfined -e CONTAINERD_ROOT=/var/lib/containerd-test -v /var/lib/containerd-test:/var/lib/containerd-test:shared -v ~/.docker/config.json:/root/.docker/config.json:shared ${{ env.REGISTRY }}/${{ env.ORGANIZATION }}/nydus-snapshotter:local
+          # start containerd
+          sudo /usr/local/bin/containerd --config /etc/containerd/containerd-test-config.toml -l debug &
+          # wait for containerd to start up
+          sleep 10
+          echo "Pull nydus-latest image"
+          sudo crictl pull ghcr.io/dragonflyoss/image-service/ubuntu:nydus-latest
+          docker logs `docker ps |grep  "nydus-snapshotter"|cut -d ' ' -f1`
+          echo "create new pod with nydus snapshotter"
+          sudo crictl run misc/example/container.yaml misc/example/pod.yaml
+          container=$(sudo crictl ps -q)
+          echo "check container liveness"
+          sudo crictl exec $container ls
+          echo "delete pod"
+          sudo crictl rmp -fa
+      - name: Cleanup containerd and snapshotter
+        if: ${{ github.event_name == 'pull_request' }}
+        run: |
+          echo "stop snapshotter"
+          docker stop snapshotter
+          docker rm snapshotter
+          echo "stop containerd"
+          sudo kill -9 `ps aux|grep "/usr/local/bin/containerd"|grep -v grep|awk -F ' ' '{print $2}'`
+          echo "delete containerd dir"
+          sudo umount -f /var/lib/containerd-test/io.containerd.snapshotter.v1.nydus/mnt
```

Comment:
After the snapshotter container is stopped, it seems that Nydud doesn't umount it, so we need to umount it here, otherwise, you cannot delete this directory.  Frankly, I don't know why Nydud didn't clean up these resources.

Reply:
Snapshotter calls nydusd to umount only after the snapshot has been cleaned up (e.g. the image is deleted).